manifests/art.yaml

```diff
@@ -2,15 +2,15 @@ updates:
   - file: "{MAJOR}.{MINOR}/elasticsearch-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
     update_list:
     # replace metadata.name value
-    - search: "elasticsearch-operator.v{MAJOR}.{MINOR}.2"
+    - search: "elasticsearch-operator.v{MAJOR}.{MINOR}.0"
       replace: "elasticsearch-operator.{FULL_VER}"
-    # replace entire version line, otherwise would replace 4.1.0 anywhere
-    - search: "version: 4.1.2"
+    # replace entire version line, otherwise would replace {MAJOR}.{MINOR}.0 anywhere
+    - search: "version: {MAJOR}.{MINOR}.0"
       replace: "version: {FULL_VER}"
-    - search: 'olm.skipRange: ">=4.1.0 <4.1.2"'
+    - search: 'olm.skipRange: ">=4.1.0 <{MAJOR}.{MINOR}.0"'
```

Comment:
same question as on the other PR...or does skipRange mean skip everything up to but not including the desired version?

Reply:
it means "this CSV can be used to replace/upgrade anything that falls in this range".  Since we don't want it to claim it can replace/upgrade itself, we use less-than